verilog/tools/formatter/verilog_format.cc

```diff
@@ -240,8 +241,20 @@ int main(int argc, char** argv) {
     return 1;
   }
 
-  if (!lines_to_format.empty() && file_args.size() > 2) {
-    std::cerr << "Note, applying --lines rules to all files" << std::endl;
+  // Some sanity checks if multiple files are given.
+  if (file_args.size() > 2) {
+    if (!lines_to_format.empty()) {
+      std::cerr << "Providing --lines but gave multiple different files."
+                << std::endl;
+      return 1;
+    }
+
+    if (!absl::GetFlag(FLAGS_inplace)) {
+      // Dumping all to stdout doesn't really make sense.
+      std::cerr << "Multiple files given. Use --inplace "
+          "for a meaningful handling of these." << std::endl;
```

Comment:
suggest: "--inplace is required when processing multiple files."

Reply:
Sounds better. Done.